### PR TITLE
FUSEDOC-818 Incorporated review comments.

### PR DESCRIPTION
**Pull request by nshendye**
Added missing template parameters.

### CODE REVIEW

File: using_images/xpaas_images/a_mq.adoc [hunk 1]
```diff
@@ -78,6 +78,10 @@ configured:
   non-containerized JBoss A-MQ, you would specify the password in the
   *_AMQ_HOME/opt/user.properties_* file. If no value is specified, a random
   password is generated.
+`*AMQ_ADMIN_USERNAME*`::
+  The username used as an admin authentication to the broker. If no value is specified, a random user name is generated.
```
Comment (adellape) [inline using_images/xpaas_images/a_mq.adoc]: s/username/user name/

Per IBM and RH Style Guides.

File: using_images/xpaas_images/a_mq.adoc [hunk 2]
```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
 *_conf_* directory in the A-MQ root and its contents used to configure the
 broker.
 
+== Configuring the JBoss A-MQ Persistance Image
```
Comment (adellape) [inline using_images/xpaas_images/a_mq.adoc]: s/Persistance/Persistence/

However, the nomenclature is "persistent" storage/volume/image. Looks like the AMQ image in question is also called that, specifically, too:

```
# oc get templates -n openshift | grep amq
amq6                                    Application template for ActiveMQ brokers.                                         12 (2 blank)      8
amq6-persistent                         Application template for ActiveMQ brokers using persistent storage.                13 (2 blank)      9
eap6-amq-persistent-sti                 Application template for EAP 6 A-MQ applications with persistent storage buil...   22 (7 blank)      11
eap6-amq-sti                            Application template for EAP 6 A-MQ applications built using STI.                  21 (7 blank)      10
```

Also [upstream](https://github.com/jboss-openshift/application-templates/tree/master/amq).

File: using_images/xpaas_images/a_mq.adoc [hunk 3]
```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
...
+
+[[configuring-params]]
+=== Application Template Parameters
+
+Basic configuration of the JBoss A-MQ Persistance xPaaS image is performed by specifying
```
Comment (adellape) [inline using_images/xpaas_images/a_mq.adoc]: s/Persistance/Persistent/

File: using_images/xpaas_images/a_mq.adoc [hunk 4]
```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
...
+  protocol must be specified, while other protocols can be optionally specified
+  as well.
+`*MQ_QUEUES*`::
+  Comma-separated list of queues available by default on the broker on its
+  startup.
+`*MQ_TOPICS*`::
+  Comma-separated list of topics available by default on the broker on its
+  startup.
+`*VOLUME_CAPACITY*`::
+  The size of the persistence storage for database volumes.
```
Comment (adellape) [inline using_images/xpaas_images/a_mq.adoc]: s/persistence/persistent/

File: using_images/xpaas_images/a_mq.adoc [hunk 5]
```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
...
+  non-containerized JBoss A-MQ, you would specify the user name in the
+  *_AMQ_HOME/opt/user.properties_* file. If no value is specified, a random user
+  name is generated.
+`*MQ_PASSWORD*`::
+  The password used for authentication to the broker. In a standard
+  non-containerized JBoss A-MQ, you would specify the password in the
+  *_AMQ_HOME/opt/user.properties_* file. If no value is specified, a random
+  password is generated.
+`*AMQ_ADMIN_USERNAME*`::
+  The username used as an admin authentication to the broker. If no value is specified, a random user name is generated.
```
Comment (adellape) [inline using_images/xpaas_images/a_mq.adoc]: s/username/user name/

File: using_images/xpaas_images/a_mq.adoc [hunk 6]
```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
...
+  The password used for authentication to the broker. If no value is specified, a random password is generated.
+`*AMQ_SECRET*`::
+  The name of a secret containing SSL related files. If no value is specified, a random password is generated.
+`*AMQ_TRUSTSTORE*`::
+  The SSL trust store filename. If no value is specified, a random password is generated.
+`*AMQ_KEYSTORE*`::
+  The SSL key store filename. If no value is specified, a random password is generated.
+
+For more information, see 
+link:../../https://github.com/openshift/openshift-docs/blob/master/dev_guide/persistent_volumes.adoc#using-persistent-volumes[Using Persistance Volumes]
```
Comment (adellape) [inline using_images/xpaas_images/a_mq.adoc]: This link does not work. Should be:

```
link:../../dev_guide/persistent_volumes.html#using-persistent-volumes[Using Persistent Volumes].
```

Also note in the above the s/Persistance/Persistent/ and addition of the period.

**Comment by adellape** [comment]
@nshendye @vikram-redhat This still needs some work before publishing. See inline for comments.

@vikram-redhat Did you want this to make it back into the OSE 3.0 book too? I'm adding the 3.1 milestone for now, but please change it to the 3.0.z one if needed.

File: using_images/xpaas_images/a_mq.adoc (hunk 2 above)
Comment (adellape) [inline using_images/xpaas_images/a_mq.adoc]: So to be clear, I think it should be called "the JBoss A-MQ Persistent Image" here.

**Comment by vikram-redhat** [comment]
@adellape - @nshendye is away on leave now. This needs to be in for 3.0.z as this is essentially a bug fix for https://issues.jboss.org/browse/FUSEDOC-818 as per @nshendye .

**Comment by adellape** [comment]
@vikram-redhat Yes, I read through the JIRA when I saw this PR, but it didn't look like it went through any review there either. We can cherry-pick for FUSEDOC-818 after the above comments have been addressed.

**Comment by vikram-redhat** [comment]
@nshendye - when you are back, can you fix the issues mentioned?

**Comment by adellape** [comment]
This has already been picked up in the enterprise-3.1 branch, but not the enterprise-3.0 branch. So I've added the `branch/enterprise-3.0` label. However, it still needs the above follow-up edits/fixes, so I didn't add the `Next Release` milestone yet. `OSE Future` seemed appropriate until this is sorted out, then we can move it to `Next Release` to get everything cherrypicked (to both branches) accordingly.

**Pull request merged**